### PR TITLE
Create "Getting Started" kata

### PR DESCRIPTION
**Pull request by cesarzc**
This change creates the "Getting Started" kata, and introduces the `@[svg]` macro to be able to use SVG media when composing katas.

### CODE REVIEW

File: katas/content/getting_started/flip_qubit/Verification.qs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    operation CheckSolution() : Bool {
+        let solution = register => Kata.FlipQubit(register[0]);
+        let reference = register => FlipQubit(register[0]);
+        let isCorrect = CheckOperationsEquivalence(solution, reference, 1);
+
+        // Output different feedback to the user depending on whether the solution was correct.
+        if isCorrect {
+            Message("Correct!");
+            Message("Congratulation! You have solved your first exercise");
```
Comment (DmitryVasilevsky) [inline katas/content/getting_started/flip_qubit/Verification.qs:17]: It's a full sentence, probably should have a period in the end.

**Review by minestarks — APPROVED** [review]

File: katas/content/getting_started/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<style>
+.transparent {
+  background: rgba(0, 0, 0, 0.0) !important;
+}
+</style>
+<table>
+    <tbody>
+        <tr class="transparent" style="border:none;">
+            <td rowspan="2" style="border:none;">
+                @[svg]({"path": "./media/Copilot.svg"})
```
Comment (billti) [inline katas/content/getting_started/index.md:20]: Being that each image is used once and you're writing inline HTML anyway, did you need a new tag and processing logic for it? You could have just inlined it - i.e. I believe the below would be valid in markdown.

```html
  <td>
    <svg width='100' height='100'>
	  <circle cx=20 cy=20 r=10 fill='blue'>
<!-- etc.. -->
    </svg>
  </td>
```

Done now I guess, so doesn't matter.

**Review by billti — APPROVED** [review]

File: katas/content/getting_started/index.md (hunk 2 above)
Comment (cesarzc) [reply katas/content/getting_started/index.md:20]: Yes, that could have been an option. Lots of opportunities to simplify composition of katas.

Comment (cesarzc) [reply katas/content/getting_started/index.md:20]: Well, having the macro makes the markdown more readable than embedding the whole SVG.

File: npm/generate_katas_content.js [hunk 3]
```diff
@@ -459,7 +535,7 @@ function createKata(segments, kataPath, globalCodeSources) {
   }
 
   return {
-    id: kataId,
+    id: id,
```
[the remark this answers is not in the document]
Comment (cesarzc) [reply npm/generate_katas_content.js]: Updated.

**Pull request merged**